geopandas/tests/test_sindex.py

```diff
@@ -393,6 +397,121 @@ def test_query_invalid_geometry(self):
         with pytest.raises(TypeError):
             self.df.sindex.query("notavalidgeom")
 
+    # `dwithin` test
+    @pytest.mark.skipif(
+        not TEST_DWITHIN,
+        reason="Requires either Shapely 2.0 or PyGEOS 0.12, and GEOS 3.10",
+    )
+    @pytest.mark.parametrize(
+        "distance, test_geom, expected",
+        (
+            (
+                0,
+                box(9.0, 9.0, 9.9, 9.9),
+                [],
+            ),
+            # bounds don't intersect and not within distance=0
```

Comment:
It looks like the comments are _after_ the blocks they apply to?  Can you please move the comment before the block?